extension/extension.js

```diff
@@ -87,5 +87,9 @@ port.onMessage.addListener(function (message) {
 	}
 });
 
-GSC.update.init();
-GSC.notifications.restore();
+chrome.runtime.getPlatformInfo(function(info) {
+	if (info.os === 'linux') {
```

Comment:
Also we must whitelist [openbsd os](https://developer.chrome.com/extensions/runtime#type-PlatformOs)